apps/syncthing/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  syncthing:
```

Comment:
`server` would be a better name for the service, since it inherits the name of the containg folder as a prefix. So this will currently be `syncthing_syncthing`. `syncthing_server` would be preferable.